tests/app/authentication/test_invalid_token_exception.py

```diff
@@ -1,6 +1,6 @@
 import unittest
```

Comment:
This test now seems pointless?we are testing an exception that is provided by the library.

Reply:
removed